04_setup_ironic.sh

```diff
@@ -12,6 +12,8 @@ source validation.sh
 
 early_deploy_validation
 
+PODMAN_VERSION=$(podman version -f json | jq -r .Version)
+
```

Comment:
Ugh, the version output also isn't compatible, so this needs to be something like `podman version -f json | jq -r '.Version,.Client.Version|strings'`

```
$ podman version -f json | jq .
{
  "Client": {
    "APIVersion": 1,
    "Version": "2.0.5",
    "GoVersion": "go1.14.7",
    "GitCommit": "",
    "BuiltTime": "Wed Sep 23 17:18:02 2020",
    "Built": 1600877882,
    "OsArch": "linux/amd64"
  }
}
```

Reply:
Also probably best to add sudo for consistency (although I think rootless podman somewhat works now, this version command actually fails on centos7 without sudo...)

Reply:
Wow. Thank you Steve, updating.